privacyidea/static/components/token/views/token.enroll.sms.html

```diff
@@ -18,7 +18,10 @@ <h4 translate>Token data</h4>
     <label for="phone" translate>Phone number</label>
     <input type="text" id="phone"
            autofocus
+           ng-required="! form.dynamic_phone"
            class="form-control"
+           placeholder="+1234567890"
+           pattern="(\+[1-9])?\d{1,14}"
```

Comment:
I am not sure what valid phone numbers actually are or should be.

Some use blanks, dots and minus?  555.123.123
I think it would be no harm to add those?

Reply:
We could add those but we might need to think about how we save the numbers internally. The [ITU E.164](https://en.wikipedia.org/wiki/E.164) does not mention any blanks, dots or dashes. We can remove them before saving the number in the DB.
This pattern is also not ITU compliant since it allows zeroes in the number (except after a leading `+`).

Reply:
Done in 42cf88a6